Test/dafny0/ConcurrentAttribute.dfy.expect

```diff
@@ -1,5 +1,16 @@
-ConcurrentAttribute.dfy(20,25): Error: reads clause might not be empty ({:concurrent} restriction)
-ConcurrentAttribute.dfy(35,23): Error: reads clause might not be empty ({:concurrent} restriction)
-ConcurrentAttribute.dfy(46,23): Error: modifies clause might not be empty ({:concurrent} restriction)
+ConcurrentAttribute.dfy(91,10): Warning: /!\ No terms found to trigger on.
+ConcurrentAttribute.dfy(92,13): Warning: /!\ No terms found to trigger on.
+ConcurrentAttribute.dfy(97,10): Warning: /!\ No terms found to trigger on.
+ConcurrentAttribute.dfy(98,13): Warning: /!\ No terms found to trigger on.
+ConcurrentAttribute.dfy(30,25): Error: reads clause could not be proved to be empty ({:concurrent} restriction)
+ConcurrentAttribute.dfy(48,23): Error: reads clause could not be proved to be empty ({:concurrent} restriction)
+ConcurrentAttribute.dfy(51,23): Error: reads clause could not be proved to be empty ({:concurrent} restriction)
+ConcurrentAttribute.dfy(55,23): Error: reads clause could not be proved to be empty ({:concurrent} restriction)
```

Comment:
I see that you recently deleted {:concurrent} checks from the auditor and various places within the last 13 commits, however, you still have issues mentioning it. Can you please explain?

Reply:
Ah which issues, I'm not sure I follow? I intended to delete all the references to `{:concurrent}` in the auditor context since I removed that case and have replaced it with verification obligations instead, but I could have missed some.